pkg/util/kubernetes/clustername/clustername.go

```diff
@@ -23,7 +23,7 @@ var (
 	// * Lowercase letters, numbers, and hyphens only.
 	// * Must start with a letter.
 	// * Must end with a number or a letter.
-	validClusterName = regexp.MustCompile(`^[a-z]([a-z0-9\-]{0,38}[a-z0-9])?$`)
+	validClusterName = regexp.MustCompile(`^([a-z]([a-z0-9\-]{0,38}[a-z0-9])?\.)*([a-z]([a-z0-9\-]{0,38}[a-z0-9])?)$`)
```

Comment:
Can the error message on L65 also get updated to include dots?

Reply:
Done.

Reply:
That `*` makes the accepted cluster name unbounded in size does it?